_data/simple-icons.json

```
@@ -10361,7 +10361,8 @@
         {
             "title": "Unity",
             "hex": "000000",
```

```suggestion
            "hex": "FFFFFF",
```
I think white is used more prominently than black.